styleguide/derek/view-templates/leadership/_leadership.scss

```diff
@@ -0,0 +1,53 @@
+.leadership-individual {
+  @include make-sm-column(3);
+  @include make-xs-column(6);
+  margin: 20px auto;
+}
+
+.leadership-image {
+  @include center-block;
+}
+
+.leadership-imageWrapper {
+  display: none;
+}
+
+.leadership-linkWrapper {
+  margin-top: 0;
+}
+
+.leadership-attrWrapper {
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.leadership-name {
+  margin-bottom: 0;
+}
+
+.leadership-title {
+  margin: 0;
+}
+
+.leadership-link {
+  @include standard-link;
+}
+
+@media only screen and (min-width: $screen-xs-max) {
+  .leadership-individual {
+    &.center-block {
+      float: none;
```

Comment:
Since you're applying the center-block mixin to leadership-individual above it seems like we can get rid of all references to the `.center-block` class and that gets rid of any bootstrap related issues.